packages/controllers/src/snaps/SnapController.test.ts

```diff
@@ -1094,6 +1094,65 @@ describe('SnapController', () => {
     snapController.destroy();
   });
 
+  describe('getRpcMessageHandler', () => {
+    it('handlers populate the "jsonrpc" property if missing', async () => {
+      const snapId = 'fooSnap';
+      const executionEnvironmentStub =
+        new ExecutionEnvironmentStub() as unknown as WebWorkerExecutionService;
+
+      const [snapController] = getSnapControllerWithEES(
+        getSnapControllerWithEESOptions({
+          state: {
+            snaps: {
+              [snapId]: {
+                enabled: true,
+                id: snapId,
+                status: SnapStatus.running,
+              },
+            },
+          } as any,
+        }),
+        executionEnvironmentStub,
+      );
+      const handle = await snapController.getRpcMessageHandler(snapId);
+      const request = { id: 1, method: 'bar' };
+      await handle('foo.com', request);
+
+      expect(request).toStrictEqual({ id: 1, method: 'bar', jsonrpc: '2.0' });
```

Comment:
Hmm, do we really want the RPC message handler to mutate the request? 

It could also just apply the `jsonrpc` property internally: `handler(origin, {...request, jsonrpc: '2.0'})`? 🤔

Reply:
I think it would be fine to do in this instance, but we shouldn't do it as a matter of principle, so I fixed it in 8ba85f5